### PR TITLE
remove ifdef from standalone spirv VUID

### PR DESCRIPTION
**Pull request by sjfricke**
`VUID-StandaloneSpirv-None-04686` is already implemented in SPIR-V Tools and I can add the `TaskPayloadWorkgroupEXT` addition if wanted

-----

- `VUID-StandaloneSpirv-PrimitivePointIndicesEXT-07103`
- `VUID-StandaloneSpirv-PrimitiveLineIndicesEXT-07104`
- `VUID-StandaloneSpirv-PrimitiveTriangleIndicesEXT-07105`

are already found in the Built-in section for example

> VUID-PrimitivePointIndicesEXT-PrimitivePointIndicesEXT-07042
The PrimitivePointIndicesEXT decoration must be used only with the the OutputPoints Execution Mode

### CODE REVIEW

File: appendices/spirvenv.adoc
Comment (sjfricke) [inline appendices/spirvenv.adoc:428]: no need to change the VUID as it has already been updated to include `TaskPayloadWorkgroupEXT`

https://github.com/KhronosGroup/SPIRV-Tools/commit/4c456f7da67c5437a6fb7d4d20d78e2a5ae2acf2#diff-38fd3032d90cb56b9e25b2b4141235b736de1ad8efc70d02dabf2be8adaf0117R211

File: appendices/spirvenv.adoc
Comment (Tobski) [inline appendices/spirvenv.adoc:569]: this looks like it's got code: twice?

File: appendices/spirvenv.adoc
Comment (dgkoch) [inline appendices/spirvenv.adoc:432]: extra "or" from above.

File: appendices/spirvenv.adoc
Comment (Tobski) [inline appendices/spirvenv.adoc:430]: extra `or`

**Comment by oddhack** [comment]
WG will sign off once @TomOlson reviews and approves.

**Review by Tobski — CHANGES_REQUESTED** [review]
Some editorial changes needed but overall LGTM

File: appendices/spirvenv.adoc
Comment (sjfricke) [reply appendices/spirvenv.adoc:430]:
```suggestion
    Class* limited to *Uniform*, *Workgroup*, *Image*, *StorageBuffer*,
```

File: appendices/spirvenv.adoc
Comment (sjfricke) [inline appendices/spirvenv.adoc:568]:
```suggestion
    The module must: not contain both an entry point that uses the
```

**Review by TomOlson — APPROVED** [review]
LGTM - I had a grammar nit but it's already been fixed. Thanks @sjfricke!

**Comment by oddhack** [comment]
@tobias I think github may be a little confused - you don't seem to have any outstanding changes requested but your review is still showing that. If I've missed something please bring it up as a separate issue, but going ahead and signing this off for the next spec update.

**Comment by sjfricke** [comment]
@oddhack he made a PR Change request, I made the change, but I think GitHub still requires the person to manually remove the "request Change" status

**Comment by oddhack** [comment]
> @oddhack he made a PR Change request, I made the change, but I think GitHub still requires the person to manually remove the "request Change" status

Huh. I am an ever-bigger fan of proposing edits in the comments that can be clicked on to accept and then resolve the issue :-) Not sure the actual 'review' mechanism is that useful for the sort of fine-grained feedback we tend to get vs. just accepting their edits and them approving, but there doesn't seem to be a gitlab-like 'approve' button yet :-(

**Pull request merged**
